variables.tf

```diff
@@ -0,0 +1,51 @@
+variable "product" {
+  type = "string"
+}
+
+variable "location" {
+  type    = "string"
+  default = "UK South"
+}
+
+// as of now, UK South is unavailable for Application Insights
+variable "appinsights_location" {
+  type        = "string"
+  default     = "West Europe"
+  description = "Location for Application Insights"
+}
+
+variable "env" {
+  type = "string"
+}
+
+variable "application_type" {
+  type        = "string"
+  default     = "Web"
+  description = "Type of Application Insights (Web/Other)"
+}
+
+variable "tenant_id" {
+  description = "(Required) The Azure Active Directory tenant ID that should be used for authenticating requests to the key vault. This is usually sourced from environemnt variables and not normally required to be specified."
+}
+
+variable "jenkins_AAD_objectId" {
+  description = "(Required) The Azure AD object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. The object ID must be unique for the list of access policies."
+}
+
+variable "subscription" {}
+
+variable "team_name" {
+  default = "rpe"
+}
+
+variable "team_contact" {
+  default = "#rpe"
+}
+
+variable "asp_capacity" {
+  default = 2
```

Comment:
So does this means in all environments we would have 2 instances of queue or is it something else ?

Reply:
This is for the App Service Plan - so 2 instances of any app service that runs in the plan. Separate from queue.